keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -516,7 +527,8 @@
           "Keys"
         ],
         "operationId": "sign",
-        "description": "Creates a signature from a digest using the specified key.",
+        "summary": "Creates a signature from a digest using the specified key.",
+        "description": "SIGN is applicable to asymmetric and symmetric keys stored in Azure Key Vault since this operation uses the private portion of the key.",
```

Comment:
nit: You could say "The SIGN operation is applicable..." just like all other operations to maintain consistency and provide better description.